Autopilot/AttitudeManager/AttitudeManager.h

```diff
@@ -15,6 +15,7 @@
 #include "net.h"
 #include "../Common/Common.h"
 
+
```

Comment:
Same thing, nothing changed in these files, so get rid of the commit / get rid of the space.